runtime/vm/createramclass.cpp

```diff
@@ -3202,6 +3202,9 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 				}
 #endif
 
+				/* prepare the bit for J9AccClassContinuation */
+				tempClassDepthAndFlags &= ~J9AccClassIsContended;
```

Comment:
this can be moved up with the rest in 

```
				tempClassDepthAndFlags &= ~(J9AccClassHotSwappedOut | J9AccClassHasBeenOverridden | J9AccClassHasJDBCNatives | J9AccClassRAMArray | (OBJECT_HEADER_SHAPE_MASK << J9AccClassRAMShapeShift));
```